### ISSUE TITLE
Toggling between ETH and USD in Send screen should show the same amounts

### ISSUE THREAD

**Issue by hboon** [AlphaWallet/alpha-wallet-ios#2243]
1. Tap Ethereum (mainnet) in Wallet tab
2. Tap Send
3. Enter 1.01 ETH
4. Tap "ETH" to toggle to USD
5. Tap "USD" to toggle back to ETH

Expected
---
6a. 1.01

Observed
---
6b. 1.0099...

**Comment by hboon** [comment]
Need to relook at this after #2275 is merged

**Comment by vladyslav-iosdev** [comment]
@hboon i found that for some reasons function 
```
extension EtherNumberFormatter {

    /// returns Double? value from `value` formatted from `EtherNumberFormatter` with appropriate `decimalSeparator` and `groupingSeparator`
    func double(from value: String) -> Double? {

        enum Wrapper {
            static let formatter: NumberFormatter = {
                let formatter = NumberFormatter()
                formatter.numberStyle = .decimal

                return formatter
            }()
        }

        let formatter = Wrapper.formatter
        formatter.decimalSeparator = decimalSeparator
        formatter.groupingSeparator = groupingSeparator

        return formatter.number(from: value)?.doubleValue
    }
}
```
doesnt work correctly, in case if we enter amount `0.004` it returns us `4.0` value

**Comment by vladyslav-iosdev** [comment]
maybe this could appear because we receive `0.004` and in `decimalSeparator` we are using `,` separator?

**Comment by hboon** [comment]
Would be more likely to be 0 if the separator is wrong?

**Comment by vladyslav-iosdev** [comment]
very strange but check this, we typing `0.002` and in response we get `2.0`

<img width="1043" alt="Screenshot 2020-11-13 at 10 16 03" src="https://user-images.githubusercontent.com/55975226/99044990-4dc65680-2599-11eb-81dd-8e79a18f586a.png">

**Comment by hboon** [comment]
Didn't undo #2249, so this is already done.
